crates/usvg/src/tree/text.rs

```diff
@@ -562,12 +567,20 @@ impl Text {
     }
 
     /// Text converted into paths, ready to render.
-    ///
-    /// Returns `None` when the `text` build feature was disabled.
     pub fn flattened(&self) -> &Group {
         &self.flattened
     }
 
+    /// The positioned glyphs and decoration spans of the text.
+    ///
+    /// This should only be used if you need more low-level access
+    /// to the glyphs that make up the text. If you just need the
+    /// outlines of the text, you should use `flattened` instead.
+    #[cfg(feature = "text")]
```

Comment:
Aren't text nodes are disabled completely when there are no `text` feature? Or have you changed that?

Reply:
I don't think they are. Otherwise this wouldn't even compile without a cfg attribute, no? (The same for all other appearences of `Node::Text`)

https://github.com/RazrFalcon/resvg/blob/5b97397af1bc995f849b274118039afa6f4d1ee3/crates/usvg/src/tree/mod.rs#L882-L889

From what I gathered only text to path conversion is locked behind the feature gate.

Reply:
Should I change this behaviour maybe?

Reply:
Yes, Text nodes themselves are enabled, but never parsed without the `text` feature. 
I changes this behaviour multiple times and already forgot which one is the current.

I think we should change it so calling `layouted()` without the `text` feature would simply print a warning. Meaning `TextFragment` should be public as well.

The `flattened()` comment is also outdated...

```rust
    /// Text converted into paths, ready to render.
    ///
    /// Returns `None` when the `text` build feature was disabled.
    pub fn flattened(&self) -> &Group {
        &self.flattened
    }
``` 

It will never return `None` anymore. The text node would not be parsed.

In a nutshell, no `text` feature - no `text` elements parsing. This is because we _must_ know bboxes of all nodes, which is impossible for text without layout/flattening.

Reply:
> Meaning TextFragment should be public as well.

I actually did that initially. The problem is that a positioned glyph contains a glyph ID (which is from `ttf-parser`, which is only imported when the text feature is enabled). I could probably replace this by u16, but the other problem is that each glyph also holds the ID of the font, which is part of the `fontdb` crate. Not sure whether this can be replaced easily?

Reply:
Ugh... Then let's keep it as is.

Reply:
Okay! We can always come up with a better solution later on.

> The flattened() comment is also outdated...

Yes, I noticed this as well and already updated the comment.